Minha primeira interação!

psalmxxiii opened the pull request:
Essa é a minha primeira tentativa de uso do GitHub. Estou traduzindo
essa página.
Traduzi até a linha 62 e estou agora “comitando" apenas para ver se
estou fazendo o procedimento corretamente, ok?!
Por favor, me avisem se estou fazendo correto o procedimento de "push"
para que eu possa continuar o trabalho.
Grato!

psalmxxiii committed:
Minha primeira interação!

Essa é a minha primeira tentativa de uso do GitHub. Estou traduzindo
essa página.
Traduzi até a linha 62 e estou agora “comitando" apenas para ver se
estou fazendo o procedimento corretamente, ok?!
Por favor, me avisem se estou fazendo correto o procedimento de "push"
para que eu possa continuar o trabalho.
Grato!

binhara commented:
aparentemente deu certo, pois veio o pull request

monoman commented:
Obrigado Flávio. Estou sem internet no momento para fazer a revisão
completa mas você traduziu partes que não podem ser traduzidas no cabeçalho
do post e em alguns links. Favor corrigir e amendar o seu PR.

:)

On Sun, Nov 9, 2014, 20:43 binhara notifications@github.com wrote:

> aparentemente deu certo, pois veio o pull request
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/MonoBrasil/website/pull/9#issuecomment-62323508.

psalmxxiii commented:
Boa noite, Rafael.

Você pode me mostrar/destacar por gentileza quais são as partes que não
deveriam ter sido traduzidas?

Não tenho certeza quais são elas...

A partir dessa sua observação vou procurar entender e me atentar para as
demais traduções que fizer.

Muito obrigado e lamento o transtorno.

Flávio.

Em 10 de novembro de 2014 08:19, Rafael Teixeira notifications@github.com
escreveu:

> Obrigado Flávio. Estou sem internet no momento para fazer a revisão
> completa mas você traduziu partes que não podem ser traduzidas no
> cabeçalho
> do post e em alguns links. Favor corrigir e amendar o seu PR.
> 
> :)
> 
> On Sun, Nov 9, 2014, 20:43 binhara notifications@github.com wrote:
> 
> > aparentemente deu certo, pois veio o pull request
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/MonoBrasil/website/pull/9#issuecomment-62323508.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/MonoBrasil/website/pull/9#issuecomment-62365206.

## 

_Flávio Grassi_

psalmxxiii committed:
Correção de tradução indevida no cabeçalho

Favor confirmar se era somente essa parte que não deveria ser traduzida.
Grato!

binhara commented:
No cabelhado do arquivo

---

title: Documentation
redirect_from:
- /Documentation/
- /Class_Library_Documentation/

##   - /Docs/

vc traduziu ele, dai ele se perder ...

--- título: "FAQ: Geral"-arquivo original_from:+redirect_from: -
/FAQ:_General/ ---

2014-11-10 22:22 GMT-02:00 Flávio Grassi notifications@github.com:

> Boa noite, Rafael.
> 
> Você pode me mostrar/destacar por gentileza quais são as partes que não
> deveriam ter sido traduzidas?
> 
> Não tenho certeza quais são elas...
> 
> A partir dessa sua observação vou procurar entender e me atentar para as
> demais traduções que fizer.
> 
> Muito obrigado e lamento o transtorno.
> 
> Flávio.
> 
> Em 10 de novembro de 2014 08:19, Rafael Teixeira notifications@github.com
> 
> escreveu:
> 
> > Obrigado Flávio. Estou sem internet no momento para fazer a revisão
> > completa mas você traduziu partes que não podem ser traduzidas no
> > cabeçalho
> > do post e em alguns links. Favor corrigir e amendar o seu PR.
> > 
> > :)
> > 
> > On Sun, Nov 9, 2014, 20:43 binhara notifications@github.com wrote:
> > 
> > > aparentemente deu certo, pois veio o pull request
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > https://github.com/MonoBrasil/website/pull/9#issuecomment-62323508.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/MonoBrasil/website/pull/9#issuecomment-62365206.
> 
> ## 
> 
> _Flávio Grassi_
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/MonoBrasil/website/pull/9#issuecomment-62481502.

## 

Alessandro de Oliveria Binhara
Azuris CloudSystem
MonoBrasil Project Founder
Xamarin Consulting Partner
Hortonworks - SI/CP Program Partner
Novo Fone: (41) 4042-0687

psalmxxiii committed:
Correção de tradução indevida no cabeçalho

Agora acho que está correto.

monoman commented:
Vou mergear e corrigir, mas a regra geral é o que vem antes do ':' (dois pontos) não pode ser traduzido, pois são 'comandos' para o engine Jekyll

psalmxxiii commented:
Obrigado pelos esclarecimentos, pessoal.

Vou me atentar para esse detalhe daqui para frente.

Flávio.

2014-11-11 8:19 GMT-02:00 Rafael Teixeira notifications@github.com:

> Merged #9 https://github.com/MonoBrasil/website/pull/9.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/MonoBrasil/website/pull/9#event-191317737.

## 

_Flávio Grassi_
